test/ejabberd_tests/tests/amp_SUITE.erl

```diff
@@ -577,14 +585,16 @@ drop_deliver_to_offline_user_test(Config) ->
               client_sends_message(Alice, Msg),
 
               % then
-              case lists:member(Rule, Rules) orelse ?config(offline_storage, Config) /= offline_failure of
+              case lists:member(Rule, Rules) orelse
+                         ?config(offline_storage, Config) /= offline_failure of
```

Comment:
According to Vim's indentation `?config` should start in the same column as `lists` in the line above.